ts/Core/Axis/BrokenAxis.ts

```diff
@@ -334,11 +343,18 @@ namespace BrokenAxis {
                             )) {
                                 eventName = 'pointInBreak';
                             }
+
                             if (eventName) {
                                 fireEvent(axis, eventName, { point, brk });
                             }
                         }
                     });
+
+                    breaksOut?.forEach(
+                        function (brk: AxisBreakOptions | undefined): void {
+                            fireEvent(axis, 'pointBreakOut', { point, brk });
```

Comment:
Looks and works well but I would stick to the naming we already have and change it to something like `pointOutsideOfBreak`